### ISSUE TITLE
TSL/ATS is not created and validated anymore during metadata import

### ISSUE THREAD

**Issue by andre-hohmann** [kitodo/kitodo-production#5207]
### Problem
During the metadata import to create a new process, the `TSL/ATS` is neither created nor validated anymore. It is possible to create the process, but after opening the process in the metadata editor, the validation message is shown: 
- Fehlende Eingabe für Monograph "null": TSL/ATS.

### Solution
As described in [Vorgang Anlegen - Monografien](https://github.com/kitodo/kitodo-production/wiki/Monografien), the `TSL/ATS` should be created automatically - as it had worked in one of the versions before. 
Furthermore, a missing `TSL/ATS` should be validated during the import process and not only in the metadata editor. 

This is probably related to the following issue, because both has worked and stopped working at the same time.  
- #5208

**Comment by matthias-ronge** [comment]
In the ruleset, there are the attributes `use="authorLastName"`, `use="title"`, `use="processTitle"`. When fixing this issue, it should be done in a way that the values of the metadata entries marked `use="authorLastName"` and `use="title"` should be used as the source for the ATS/TSL creation, and the generated ATS/TSL should be written into any metadata fields tagged `use="processTitle"` (—a misnomer, by the way, that should be renamed one day, but that’s worth a separate ticket—).

**Comment by markusweigelt** [comment]
@matthias-ronge I don't think that is part of this issue. Cause this issue is a bug and your suggestion is more a feature. So i think it is better to create an extra issue for that to fix it seperatly.

**Comment by andre-hohmann** [comment]
@matthias-ronge : Maybe, the following issue could be solved with your approach:
- #4461

I agree with @markusweigelt that this should be done separately.

**Comment by markusweigelt** [comment]
Feature issue #5289 is created to replace ATS/TSL generation with more generic way to get fields from ruleset instead of using fixed field names.

**Comment by andre-hohmann** [comment]
The behavior is no longer reproducible - i close the issue.
